install_config/install/advanced_install.adoc

```diff
@@ -71,9 +71,49 @@ see the
 xref:../../scaling_performance/install_practices.adoc#scaling-performance-install-best-practices[Scaling and Performance Guide].
 
 After following the instructions in the
-xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites] topic and
-deciding between the RPM and containerized methods, you can continue in this
-topic to xref:configuring-ansible[Configuring Ansible Inventory Files].
+xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites]
+topic, deciding between the RPM and containerized methods and
+xref:advanced-cloud-providers[choosing from the on-premises or cloud scenarios],
+you can continue in this topic to
+xref:configuring-ansible[Configuring Ansible Inventory Files].
+
+[[advanced-cloud-providers]]
+=== Choosing clouds over on-premises
+
+Provisioning of VMs in a cloud, defining your cloud infrastructure and applying
+the required post-provision configuration tasks may be assisted with Ansible
+automation playbooks for the supported cloud providers. This advanced installation
+guide is happen to fulfill that purpose as well.
+
+==== OpenStack provider
+
+In order to install {product-title} with manual steps using OpenStack CLI,
+see the
+link:https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/deploying_and_managing_red_hat_openshift_container_platform_3.6_on_red_hat_openstack_platform_10[reference architecture],
+which is actual for {product-title} 3.6 and Red Hat OpenStack Platform 10.
+
+As a prerequisite, you will have to provision VMs and configure the cloud
+infrastructure, like network, storage, firewall and security groups.
+Some of the basic configuration may be assisted by that reference architecture
+and the
+xref:../../install_config/install/prerequisites#prereq-cloud-provider-considerations[cloud provider considerations].
+For advanced configuration prerequisites, like DNS, load balancing and high
+availability, you may want to use the
+link:https://github.com/openshift/openshift-ansible-contrib/tree/master/playbooks/provisioning/openstack[Ansible automation playbooks]
+instead. For automated installations, see also
+xref:../../install_config/configuring_openstack#install-config-configuring-openstack[Configuring for OpenStack]
+and
+xref:configuring-ansible[Configuring Ansible Inventory Files].
+
+====
+[IMPORTANT]
+The reference architecture for automated installations based on
+link:https://docs.openstack.org/heat/latest[OpenStack Heat] templates for
+link:https://access.redhat.com/documentation/en-us/reference_architectures/2017/html/deploying_red_hat_openshift_container_platform_3.4_on_red_hat_openstack_platform_10[{product-title} (OCP 3.4) on Red Hat OpenStack Platform 10]
+is deprecated. For OSP 13 time frame, it is being replaced with the
```

Comment:
I'm not sure it's been officially deprecated. More like it's not being actively developed anymore.

Reply:
I'm not sure the docs should state something is not being actively developed anymore :)

Reply:
let it be 'unsupported'

Reply:
done